src/py/aspen/app/app.py

```diff
@@ -5,9 +5,16 @@
 
 from authlib.integrations.flask_client import OAuth
 from flask import redirect, session
+from flask_cors import CORS
 
 from aspen.app.aspen_app import AspenApp
-from aspen.config import config, DevelopmentConfig, ProductionConfig, StagingConfig
+from aspen.config import (
+    config,
+    DevelopmentConfig,
+    LocalConfig,
```

Comment:
is there a reason to keep around `DevelopmentConfig`?

Reply:
Right now we're setting up an alternate development path -- the old setup should continue to work for the most part, and each engineer can switch over to dockerized local dev at their own pace. Once we're happy with dockerized dev, we can remove the old configs and docs.

Reply:
Got it.